apps/trading/components/welcome-dialog/telemetry-approval.tsx

```diff
@@ -0,0 +1,26 @@
+import { Checkbox } from '@vegaprotocol/ui-toolkit';
+import { t } from '@vegaprotocol/i18n';
+import { useTelemetryApproval } from '../../lib/hooks/use-telemetry-approval';
+
+export const TelemetryApproval = () => {
+  const [isApproved, setIsApproved] = useTelemetryApproval();
+  return (
+    <div className="flex flex-col px-2 py-3">
+      <div className="mr-4" role="form">
+        <Checkbox
+          label={<span className="text-lg pl-1">{t('Share usage data')}</span>}
+          checked={isApproved}
+          name="telemetry-approval"
+          onCheckedChange={() => setIsApproved(!isApproved)}
+        />
+      </div>
+      <div className="text-xs text-neutral-500 dark:text-neutral-400 ml-6">
+        <span>
+          {t(
+            'Help identify bugs and improve the service by sharing anonymous usage data. you can change this in your settings at any time.'
```

Comment:
@macqbat This text needs to be different for the risk dialog and the settings page as it doesnt make sense in the context of the settings page, can you make this a prop

Reply:
done